lib/src/config_handler.dart

```diff
@@ -109,6 +116,28 @@ class ConfigHandler {
     }
     configFile.writeAsStringSync(newTool, mode: FileMode.append);
   }
+
+  /// Disables the reporting capabilities if true is passed
+  void enableTelemetry(bool reportingBool) {
+    final String flag = reportingBool ? '1' : '0';
+    final String configString = configFile.readAsStringSync();
+
+    final RegExp regex = RegExp(disableTelemetryPattern, multiLine: true);
```

Comment:
can you pull compiling this regex into a static variable?

Also does this have to be multiLine: true? Could you split the config by newlines and then test the regex on each line iteratively? It's more difficult for me to reason about multiline regexes.

Reply:
Yes I can make the regex object a static variable, and in this case, I believe multiline would be best because having multiple matches would be an error that would prompt a reset of the config file.

Doing this iteratively can be done as well but I didn't want to use a counter variable for number of matches.

Reply:
Also the regex string pattern starts with the `^` character and ends with `$` which signify the start and end of a line so we shouldn't have a problem with matches that span more than one line

Reply:
SGTM